analyze/with-metabase.md

```diff
@@ -8,81 +8,31 @@ As [Metabase](https://www.metabase.com/) puts it:
 
 > **Business intelligence for everyone** Team doesn’t speak SQL? No problem. Metabase is the easy, open-source way to help everyone in your company work with data like an analyst.
 
-### Run Metabase and Hydra using Docker Compose
+## Setup
 
-Copy the following `docker-compose.yaml` file to your local machine.
+- Ensure you have the details of your Hydra instance handy.
+- Sign up for [Metabase](https://www.metabase.com/).
 
-```
-version: '3.9'
-services:
-  postgres:
-    image: postgres:14
-    restart: always
-    volumes:
-    - pg_data:/var/lib/postgresql/data
-    environment:
-      POSTGRES_USER: &metabase_user metabase
-      POSTGRES_PASSWORD: &metabase_password metabasepw
-      POSTGRES_DB: &metabase_db metabase
-  hydra:
-    platform: linux/amd64
-    image: ghcr.io/hydrasdb/hydra
-    restart: always
-    volumes:
-      - hydra_data:/home/postgres/pgdata
-    environment:
-      PGPASSWORD_SUPERUSER: hydra
-    ports:
-    - "6432:5432"
-  metabase:
-    image: metabase/metabase:v0.44.2
-    ports:
-    - "3000:3000"
-    depends_on:
-    - postgres
-    - hydra
-    restart: always
-    environment:
-      MB_DB_TYPE: postgres
-      MB_DB_USER: *metabase_user
-      MB_DB_PASS: *metabase_password
-      MB_DB_DBNAME: *metabase_db
-      MB_DB_HOST: postgres
-      MB_DB_POST: 5432
-volumes:
-  pg_data:
-  hydra_data:
-```
-
-This compose file defines three services: Metabase, a Hydra instance, and an isolated Postgres instance for storing Metabase’s metadata.
-
-#### Seed some sample data
+### Seed some sample data
 
-Now, let’s start by creating a sample database in Hydra and populate it with some sample data.
+Let’s start by creating a sample table in Hydra and populate it with some data. You'll need to replace your Hydra connection string or set the `PGCONN` environment variable.
 
-* In a terminal from the folder where you’ve saved the `docker-compose.yaml` file:
-  * Start Hydra with `docker compose up hydra`
-  * Wait for the Hydra instance to complete first-time provisioning by looking for the log line `initialized a new cluster`
-*   In another terminal let’s create a sample database and seed some data. You will be prompted for the Hydra password twice. It is (as configured in compose) `hydra`.
+* Create a sample database and seed some data. You will be prompted for the Hydra password twice. It is (as configured in compose) `hydra`.
 
     ```
-    createdb -h localhost -p 6432 -U postgres sample_data
-    psql -h localhost -p 6432 -U postgres -d sample_data \
-        -c "CREATE TABLE data (id uuid NOT NULL PRIMARY KEY DEFAULT gen_random_uuid(), sample integer, timestamp timestamptz) USING columnar;" \
-      -c "INSERT INTO data (sample, timestamp) SELECT floor((random() + random() + random() + random() + random() + random()) / 6 * 100000000)::int, to_timestamp(EXTRACT(epoch from NOW()) - floor(random() * 2600000)) FROM generate_series(1, 50000);"
+    psql $PGCONN \
```

Comment:
```suggestion
    psql "$PGCONN" \
```

Reply:
you can skip these if you've confirmed it works without quotes, I _think_ I've run into issues not using quotes before but not 100% sure.

Reply:
Its a good idea incase someone is using a weird shell.